src/core/elements/demo_solid_box.rs

```diff
@@ -0,0 +1,45 @@
+use crate::prelude::*;
+
+#[derive(HasSchema, Default, Debug, Clone)]
+#[type_data(metadata_asset("demo_solid_box"))]
+#[repr(C)]
+pub struct DemoSolidBoxMeta {
+    pub atlas: Handle<Atlas>,
```

Comment:
Regarding the way that we define solids, I'm pretty sure we don't want to add the new element metadata to the map. Instead we should put all of the metadata for the solids in whatever kind of elements that we want to spawn.

For instance, now that we have the physics implemented, any element, such as a "door" element, can spawn it's own entities with a `Solid` component to serve as a collision, while also spawning whatever sprites it needs with different sizes, etc.

So all of the spawning logic would be implemented by the door, instead of putting any of that info in the map metadata.

Does that make sense for what you were thinking?